src/styles/less/MissionControl.less

```diff
@@ -6,47 +6,70 @@
   height: 70vh;
 }
 
```

Comment:
All of these changes take place in the appropriate style file for the page in question. Manipulation in this file manipulates the layout of the elements on child/missioncontrol